cha/req.tex

```diff
@@ -0,0 +1,89 @@
+\section{Requirement Elicitation}
+To do requirement elicitation, we focus on the requirements posed in \cref{part:introduction}.
+These requirements will be elicitated and explained, to get an overview of the requirements for the app.
+
+The elicitated requirements will be divided into two categories: functional requirements and UI/UX requirements.
+We start with describing the technical requirements which can be elicitated from \cref{part:introduction}.
+
+\begin{eletterate}
+    \item \textbf{Run on Android 5.0+} \hfill\\

```

Comment:
Should be something like:

The Android sound stack underwent significant design changes in version 5.0. We determine those changes might be important for our design and implementation.